app/views/application/_panel_flash_messages.html.slim

```diff
@@ -17,15 +17,15 @@
 - [:alert, :notice, :warning].each do |name|
   - {alert: 'icon_circled_i', warning: 'icon_circled_x'}[name].tap do |icon_partial|
     - if flash[name]
-      div class="single-panel--content single-panel--content--#{name} icon-and-text--wrapper"
+      .single-panel--content.icon-and-text--wrapper class="single-panel--content--#{name}"
```

Comment:
This also changes the order of application for these classes, and possibly the precedence of some styles.

AFAICT it should not make a difference in this case, but it seems wise for a developer to be sure they know how the classes interact before swapping the order simply for code readability.

Reply:
This is a good observation. Should we encounter a case where the classes must be applied in a certain order, that could certainly trigger an exception to this convention. I'd also consider that a code smell that something fishy is going on with the CSS (as I would consider the need to apply classes in a particular order to be a CSS anti-pattern).